### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

### CODE REVIEW

**Review by MarcoGorelli — COMMENTED** [review]
thanks for working on this - looks like there's a test failure
```
FAILED pandas/tests/scalar/test_nat.py::test_nat_doc_strings[Timedelta.total_seconds] - AssertionError
```

File: pandas/core/groupby/generic.py [hunk 1]
```diff
@@ -914,6 +914,25 @@ def fillna(
...
+
+        >>> lst = ['a', 'a', 'b', 'b']
+        >>> ser = pd.Series([1, np.nan, 3, np.nan], index=lst)
+        >>> ser
+        a    1.0
+        a    NaN
+        b    3.0
+        b    NaN
+        dtype: float64
+        >>> ser.groupby(level=0).fillna(value=0)
```
Comment (MarcoGorelli) [inline pandas/core/groupby/generic.py:930]: this one works, but isn't how one would usually use `groupby.fillna` - for this example, you could just do `ser.fillna(0)`

how about
```python
ser.groupby(level=0).fillna(method='ffill')
```
to better show what's unique about this method?

**Review by MarcoGorelli — COMMENTED** [review]
Nice! Just got a comment on groupby.fillna, rest looks good

File: pandas/core/groupby/generic.py (hunk 1 above)
Comment (DeaMariaLeon) [reply pandas/core/groupby/generic.py:930]: I thought that the keyword `method` there was deprecated https://github.com/pandas-dev/pandas/pull/53496

**Review by MarcoGorelli — APPROVED** [review]
nice one, thanks @DeaMariaLeon !

File: pandas/core/groupby/generic.py (hunk 1 above)
Comment (MarcoGorelli) [reply pandas/core/groupby/generic.py]: ah you're right, thanks!

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @DeaMariaLeon

**Pull request merged**